data/simulations/FAH_SARS-CoV-2_Mpro_FEP.yml

```diff
@@ -0,0 +1,57 @@
+type: md
+title: Trajectories of the SARS-CoV-2 main protease protein from Folding@Home
+description: >
+  All-atom MD simulations of the SARS-CoV-2 main protease with small molecules, simulated using Folding@Home.
```

Comment:
"with small molecules from the COVID Moonshot"?

Don't forget you can include links, just as if this were markdown!